bower.json

```diff
@@ -32,15 +32,14 @@
     "url": "git://github.com/patternfly/angular-patternfly.git"
   },
   "dependencies": {
-    "angular": "1.2.25 - 1.3.*",
-    "patternfly": "~2.0.0",
-    "c3": "~0.4.10"
```

Comment:
Since base patternfly isn't set up to use bower properly it will come down twice. They are checking in the libraries directly to the components directory. So we will get it once from the components directory and a second time from this bower dependency include. We don't have dependencies for the other libraries (datepicker) so I would suggest we add all of them or just let PF reference handle it.